components/org.wso2.carbon.metrics.core/src/main/java/org/wso2/carbon/metrics/core/Metrics.java

```diff
@@ -47,6 +47,7 @@
 public class Metrics {
 
     private static final Logger logger = LoggerFactory.getLogger(Metrics.class);
+    private static final String STREAMLINED_METRICS_NAMESPACE = "metrics";
```

Comment:
Can't we avoid defining the constants here? ConfigProvider should figure out where to get configs.

Reply:
@niveathika @mohanvive FYA